<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core::Internal::PageWidget</class>
 <widget class="QWidget" name="Core::Internal::PageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="Utils::MiniSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="pageTree">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <column>
       <property name="text">
        <string>0</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="layout">
       <property name="spacing">
        <number>-1</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedPages">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>250</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::MiniSplitter</class>
   <extends>QSplitter</extends>
   <header location="global">utils/widgets/minisplitter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
